java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -560,4 +560,36 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<ClusterValue<String>> lolwut(int version, int[] parameters, Route route);
+
+    /**
+     * Returns the number of keys in the database.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @see <a href="https://redis.io/docs/latest/commands/dbsize/">redis.io</a> for details.
```

Comment:
can you update all links to ValKey?

Reply:
oh sure, thought we were updating all of them later